kiwi/bootloader/config/grub2.py

```diff
@@ -1114,9 +1114,7 @@ def _create_early_boot_script_for_mbrid_search(self, filename, mbrid):
                 'set btrfs_relative_path="yes"{0}'.format(os.linesep)
             )
             early_boot.write(
-                'search --file --set=root /boot/mbrid{0}'.format(
-                    os.linesep
-                )
+                f'search --file --set=root /boot/{mbrid.get_id()}{os.linesep}'
```

Comment:
To clarify here, I think we should have a prefix, because goopy hex is not particularly clear for what it is.

That is, `mbrid-{mbrid.get_id()}{os.linesep}` instead.

Reply:
The file doesn't end in the system and is only an identifier that is stored on the media (the ISO). It's imho not something that needs to be understood by people and it helps the debugging if the name of the file represents the hex value that you can find in the binary structure of the MBR. Actually I don't want to change that as we have this meta information on the media for years and I don't see a value in changing the name

> What does this look like as an example?

if you look on a kiwi install ISO by mounting it you will find files like this:

```
ls -l /mnt/boot/
total 5
-rw-r--r-- 1 root root   11 Nov  7 16:21 0x5c4003f2
drwxr-xr-x 1 root root 2048 Nov  7 16:21 grub2
-rw-r--r-- 1 root root   11 Nov  7 16:21 mbrid
drwxr-xr-x 1 root root 2048 Nov  7 16:21 x86_64
```

The earlyboot script ran the lookup against /boot/mbrid but this is not unique, with this change the lookup happens against the 0xID which is then unique to the media.

Reply:
Okay then.